layouts/partials/css.html

```diff
@@ -14,4 +14,4 @@
 {{- $css := resources.Get $sass | toCSS $cssOpts | minify | fingerprint }}
 <link rel="stylesheet" type="text/css" href="{{ $css.RelPermalink }}" integrity="{{ $css.Data.Integrity }}">
 {{- end }}
-<link rel="stylesheet" type="text/css" href="{{ "css/icons.css" | absURL }}">
+<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css" integrity="sha384-KA6wR/X5RY4zFAHpv/CnoG2UW1uogYfdnP67Uv7eULvTveboZJg0qUpmJZb5VqzN" crossorigin="anonymous">
```

Comment:
Is it better to use a CDN instead of bundling it with the website?

Reply:
I'm not sure, honestly. My rationale was this: downloading from a trusted third party seems like it would reduce the possibility of anything malicious being downloaded and _seems_ to make updating things in the future easier because we just need to change a URL link instead of downloading a specific file and renaming it to icons.css If you would prefer to download the file and include it as part of the site that's fine by me. I just chose one path when I think there are several good ways to achieve it :)